### PR TITLE
feat(email): update newsletter strings

### LINKED ISSUE #5460: Update Copy Around Newsletter Opt-in
From Ben Niolet (and retention marketing team): 

"TL;DR: We would like to change the language on email opt-ins on the accounts sign up form and in the settings UI for Accounts. This is so we don't confuse users when we start sending Account-related email that isn't opt-in. 

(...)

We have worked up some draft copy and would like to work with you to get this implemented. Additionally, we'd like to change the Account sign-up form email opt-in language. It currently refers to a monthly newsletter, but we have moved away from monthly newsletters (*and newsletters in gerneral*) and think we have some more accurate language. 

The copy we worked up and the existing experience for comparison is here: 

https://docs.google.com/a/mozilla.com/document/d/1jzL4yM_JafGN1OmHPGaAmtasJxHMlkaYumbL1g88Fo4/edit?usp=sharing

"

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #5460

@vbudhram r?

### CODE REVIEW

File: app/scripts/templates/settings/communication_preferences.mustache [hunk 1]
```diff
@@ -23,7 +23,10 @@
 
     <form novalidate>
       <p>
-        {{#t}}Receive the monthly Firefox newsletter.{{/t}}
+        {{#t}}We may send you emails related to your Firefox Account.{{/t}}<br/>
+
+        {{#t}}Additionally, you can choose whether to receive the latest news about Mozilla and Firefox.{{/t}}
```
Comment (vbudhram) [inline app/scripts/templates/settings/communication_preferences.mustache]: Text looks a little tight. Maybe another line break or some padding? @ryanfeeley thoughts?

screen shot 2017-09-21 at 3 03 38 pm

**Comment by ryanfeeley** [comment]
@vbudhram @vladikoff Probably makes sense to put re-order this so that the latest news is first. Something like:
* Receive the latest news about Mozilla and Firefox, beyond the emails we send related to activity in your Firefox Account.

**Comment by vladikoff** [comment]
@ryanfeeley Ryan, this is not the confirmed copy in https://docs.google.com/document/d/1jzL4yM_JafGN1OmHPGaAmtasJxHMlkaYumbL1g88Fo4/edit

**Comment by ryanfeeley** [comment]
@vladikoff They took my other suggestion. Maybe they'll take this one. I commented in the doc.

**Comment by davismtl** [comment]
I don't fully agree with @ryanfeeley 's recommended changes.

Please, let's just land the strings from the document as they were originally agreed on. They need it up ASAP because I think there is something relating to legal.

We can make iterations after. They're simple changes.

**Comment by vladikoff** [comment]
Thanks @vbudhram !

**Pull request merged**